lib/improver/wind_downscaling.py

```diff
@@ -445,20 +449,21 @@ def _interpolate_1d(xup, xlow, at_x, yup, ylow):
         """Simple 1D linear interpolation for 2D grid inputs level.
 
         Args:
-            xup: 2D np.array (float32)
-                upper x-bins
-            xlow: 2D np.array (float32)
-                lower x-bins
-            at_x: 2D np.array (float32)
+            xup (np.ndarray):
+                2D array float32 - upper x-bins
+            xlow (np.ndarray):
+                2D array float32 - lower x-bins
+            at_x (np.ndarray):
```

Comment:
It might be worth adding to the sentence below that this is expected to be a 2D array, for consistency with the other docstrings. This applies at `at_x`, `yup` and `ylow`